src/components/AppHeader/AppHeader.css

```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```

Comment:
I'd love to see us move away from pixels across the board (better for accessibility), but that's a much longer discussion, and one I'll probably lose. :)